kart/completion.py

```diff
@@ -114,6 +141,39 @@ def install_fish(*, prog_name: str, complete_var: str, shell: str) -> Path:
     return path_obj
 
 
+def install_powershell(*, prog_name: str, complete_var: str, shell: str) -> Path:
+    subprocess.run(
+        [
+            shell,
+            "-Command",
+            "Set-ExecutionPolicy",
+            "Unrestricted",
+            "-Scope",
+            "CurrentUser",
+        ]
+    )
+    result = subprocess.run(
+        [shell, "-NoProfile", "-Command", "echo", "$profile"],
+        check=True,
```

Comment:
check=True means that subprocess.run raises an error if the exit-code is non-zero - either check the return code yourself, or use check=True, but there's no point doing both at once